lib/core/package.json

```diff
@@ -47,7 +47,7 @@
     "commander": "^2.19.0",
     "common-tags": "^1.8.0",
     "core-js": "^2.5.7",
-    "css-loader": "^1.0.0",
+    "css-loader": "^1.0.1",
```

Comment:
Looks like `css-loader` is used in some other packages too. Can you update it there as well?

Reply:
Sure thing, happy to!